homeassistant/components/starlink/sensor.py

```diff
@@ -122,6 +123,22 @@ def native_value(self) -> StateType | datetime:
         native_unit_of_measurement=PERCENTAGE,
         value_fn=lambda data: data.status["pop_ping_drop_rate"] * 100,
     ),
+    StarlinkSensorEntityDescription(
+        key="upload",
+        translation_key="upload",
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfInformation.BYTES,
+        value_fn=lambda data: data.usage["upload_usage"],
+    ),
+    StarlinkSensorEntityDescription(
+        key="download",
+        translation_key="download",
```

Comment:
Not sure if I appropriate key used and if so are the translation keys needed in this case?

Reply:
you'll still need the translation keys as they have the same device class and therefore would get the same name